pkg/hub/submarinerbroker/controller.go

```diff
@@ -115,9 +131,57 @@ func (c *submarinerBrokerController) sync(ctx context.Context, syncCtx factory.S
 	return c.createIPSecPSKSecret(config.SubmarinerNamespace)
 }
 
+func buildBrokerConfig(clusterSet *clusterv1beta1.ManagedClusterSet) (*brokerConfig, error) {
+	config := &brokerConfig{
+		SubmarinerNamespace:         brokerinfo.GenerateBrokerName(clusterSet.Name),
+		Components:                  []string{"service-discovery", "connectivity"},
+		DefaultCustomDomains:        nil,
+		GlobalnetCIDRRange:          "242.0.0.0/8",
+		DefaultGlobalnetClusterSize: 65536,
+		GlobalnetEnabled:            false,
```

Comment:
I manually modified this to "true" and wanted to verify if globalnet is getting deployed with KIND Clusters (i.e., `make images`, `make clusters` and `make demo`). However, globalnet is not deployed.

Also, broker-globalnet-cm as well as broker objects are not created on the broker cluster.  

Can you please let us know how you are testing this change?